code/api/package.json

```diff
@@ -12,7 +12,8 @@
     "start": "nodemon src/index.js --exec babel-node",
     "start:prod": "npm run build:prod && npm run start:server",
     "build:prod": "babel src -s -D -d build",
-    "start:server": "node build/index.js"
+    "start:server": "node build/index.js",
+    "test": "jest --watch"
```

Comment:
I wonder if this is something we should be putting in our pack.json in web to get it to work.